modules/nf-core/samtools/sort/tests/main.nf.test

```diff
@@ -13,15 +13,43 @@ nextflow_process {
         config "./nextflow.config"
 
         when {
-            params {
-                outdir   = "$outputDir"
+            process {
+                """
+                input[0] = Channel.of([
+                    [ id:'test', single_end:false ], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/bam/test.paired_end.bam', checkIfExists: true)
+                ])
+                input[1] = Channel.of([
+                    [ id:'fasta' ], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/genome.fasta', checkIfExists: true)
```

Comment:
do we need fasta for bam?

Reply:
not necessarily, it doesn't really matter whether it's there or not